lib/config.js

```diff
@@ -95,26 +95,35 @@ function loadConfig(filePath) {
         // a "parent". Load the referenced file and merge the configuration recursively.
         if (config.extends) {
 
-            var parentPath = config.extends;
-            if (isFilePath(parentPath)) {
-                // If the `extends` path is relative, use the directory of the current configuration
-                // file as the reference point. Otherwise, use as-is.
-                parentPath = (!isAbsolutePath(parentPath) ?
-                    path.join(path.dirname(filePath), parentPath) :
-                    parentPath
-                );
+            if (config.extends.constructor !== Array) {
```

Comment:
Should this be using `instanceof` instead? See [.constructor](https://github.com/eslint/eslint/search?p=4&q=constructor&utf8=%E2%9C%93) vs [instanceof](https://github.com/eslint/eslint/search?p=4&q=instanceof&utf8=%E2%9C%93)